docs/deployment.rst

```diff
@@ -156,5 +160,5 @@ and no other interpreter or apllication has to be deployed.
 See also
 --------
 
-* `SSH Daemon manpage<http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/sshd.8>`_
-* `Ansible<http://docs.ansible.com>`_
+* `SSH Daemon manpage <http://www.openbsd.org/cgi-bin/man.cgi/OpenBSD-current/man8/sshd.8>`_
+* `Ansible <http://docs.ansible.com>`_
```

Comment:
did you also `make -C docs html` to generate the html files with sphinx?
one can then just open them locally within a browser and look if everything looks pretty.